src/base.ts

```diff
@@ -0,0 +1,11 @@
+export interface BaseBLE {
```

Comment:
Ah I thought I commented this somewhere. But could you name this file `Ble.ts` and the interface `Ble`? Thats the last thing, I PROMISE.

Reply:
sure! and no worries. thats actually much better anyway